app/script/components/list/conversationListCell.js

```diff
@@ -35,7 +35,7 @@ z.components.ConversationListCell = class ConversationListCell {
       .computed(() => this.cell_state(z.conversation.ConversationCellState.generate(this.conversation)))
       .extend({rateLimit: 500});
 
-    this.showJoinButton = ko.pureComputed(() => this.conversation.hasActiveDeclinedCall());
+    this.showJoinButton = ko.pureComputed(() => this.conversation.hasJoinableCall());
```

Comment:
you can even remove the `ko.pureComputed(() =>` since it is already an observable and you do not do any operation on it